assignments/prototypes.js

```diff
@@ -7,8 +7,42 @@
   
   Each constructor function has unique properties and methods that are defined in their block comments below:
 */
+  const GameObject = function(attributes){
+    this.createdAt = 'June52019',
+    this.name = attributes.name,
+    this.dimensions = attributes.dimensions
+    this.destroy = function() {
+      return `${this.name} was removed from the game.`}
+  }
```

Comment:
We don't want to define the function in the constructor. When we do that, every time we make a new object it will define a new copy of that function. The advantage of a prototype is that we declare it once, and then every copy of the object will use the same prototype function